src/test/java/com/datastax/oss/driver/internal/core/loadbalancing/LoadBalancingPolicyTestBase.java

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
Same here: this looks to be nearly identical to the class [already defined](https://github.com/apache/cassandra-java-driver/blob/4.18.1/core/src/test/java/com/datastax/oss/driver/internal/core/loadbalancing/LoadBalancingPolicyTestBase.java) in cassandra-java-driver.  Do we really need this here?

Reply:
I copied and pasted both files to set up mockito stuff for the unit tests. It seems like only the unit tests in `BasicLoadBalancingPolicyQueryPlanTest` are not needed and I will delete them. Anything else you want me to delete/refactor?